### PR TITLE
Change maxWorkers default to half your CPUs

### PR DESCRIPTION
**Pull request by BelfordZ**
## Explanation

Currently the unit tests take a very long time to run locally. The maxWorkers is defaulted to 2, and on a big machine this is wasting a lot of time waiting for tests to pass. This PR makes the tests use half your cpu, and on my machine it made the tests run in roughly 1/5 the time. 

## Manual Testing Steps

Just run the tests as you normally would

## Pre-merge author checklist

- [x] I've clearly explained:
  - [x] What problem this PR is solving
  - [x] How this problem was solved
  - [x] How reviewers can test my changes
- [x] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

File: test/run-unit-tests.js [hunk 1]
```diff
@@ -133,10 +134,10 @@ async function start() {
         })
         .option('maxWorkers', {
           alias: ['mw'],
-          default: 2,
+          default: Math.max(Math.floor(cpus().length / 2), 1), // once using node 18, we should change this to use os.availableParallelism()
```
Comment (BelfordZ) [inline test/run-unit-tests.js:137]:
```suggestion
          default: Math.max(Math.floor(cpus().length / 2), 2), // once using node 18, we should change this to use os.availableParallelism()
```

**Comment by brad-decker** [comment]
pretty sure this is having unintended affects in CI

**Comment by brad-decker** [comment]
Screenshot 2023-07-19 at 12 01 07 PM

on this pr versus normal:

Screenshot 2023-07-19 at 12 02 15 PM

**Comment by BelfordZ** [comment]
A bit puzzling that it doesn't work in CI. At first I thought it was the odd number of CPUs, so i revised the code to take the ceil value of the cpus/2, but that still doesn't work in CI. 

Maybe we can set the value explicitly when running the tests in CI?

**Pull request closed without merging**